README.md

```diff
@@ -116,6 +116,15 @@ yarn prettier --check .
 yarn prettier --write .
 ```
 
+## Troubleshooting
+
+If during development changes do not seem to be taking effect, try:
+
+- Restart the application:
+- Restart the Docker container `docker-compose down && GENERATE_CONFIG=false docker-compose up -d`
```

Comment:
If you intend to destroy and recreate the container this sequence is fine, but you labelled it "restart" which would be `docker-compose restart`.  Down destroys the containers and networks.  Up creates them from the image and compose file.